packages/lib-react-components/src/components/layout/zoo-footer.jsx

```diff
@@ -9,14 +9,20 @@ export const StyledFooterBox = styled(Box)`
   border-top: 5px solid #005D69;
   font-size: 16px;
 
-  &.footer__nav {
+  .footer__nav {
     font-size: 1em;
+    min-width: 192px;
+  }
+
+  .footer__logo {
+    g {
```

Comment:
This doesn't feel like the best way to make the ZooniverseLogotype the desired color, open to suggestion.

Reply:
How about we move the `homeLogoDiv` actually into the markup, make the wrapping div a grommet `Box` and make a `StyledZooniverseLogoType` component defined using `styled(ZooniverseLogoType)` targeting its fill color? We'll still want `People Powered Research` string as a prop for translations. Does that make sense?